library/src/main/kotlin/com/freeletics/flowredux/FlowRedux.kt

```diff
@@ -42,18 +35,19 @@ fun <A, S> Flow<A>.reduxStore(
 
     coroutineScope {
 
-        val actionBroadcastChannel: BroadcastChannel<A> = this@reduxStore.broadcastIn(this)
+        val actionBroadcastChannel: BroadcastChannel<A> = BroadcastChannel(100)
```

Comment:
Not sure about what `capacity` here is actually appropriate. `broadcastIn` uses 1, but that's not enough for us.

Reply:
But do we need to broadcast more than 1 action at a time?